app/views/games/new.html.erb

```diff
@@ -1,21 +1,22 @@
 <h1>
-  <p style="float: left;">New Game</p>
+  <p style="float: left;">
```

Comment:
I see you changed this page, but provided no screens for it

Reply:
<img width="1652" alt="image" src="https://github.com/robert-damoc/game-stats/assets/137196597/5e839303-689d-4fd5-864b-0098519c06e7">